### PR TITLE
Update prepare-for-directory-synchronization.md

### PR DESCRIPTION
**Pull request by msbemba**
updated Non ASCII characters does not sync for any attributes as there was no specific mention about that in the document.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6395

File: microsoft-365/enterprise/prepare-for-directory-synchronization.md [hunk 1]
```diff
@@ -44,6 +44,9 @@ However, directory synchronization requires planning and preparation to ensure t
 
 Follow these steps in order for the best results.
 
+> [!NOTE]
+> Non ASCII characters does not sync for any attributes on the AD DS user account
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/prepare-for-directory-synchronization.md]:
```suggestion
> Non-ASCII characters do not sync for any attributes on the AD DS user account.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@LauraWi / @kelleyvice-msft Please merge this PR. Thanks

**Pull request merged**